dataRoutes.js

```diff
@@ -0,0 +1,23 @@
+const express = require("express");
+const router = new express.Router();
+const ExpressError = require("./expressError")
+const upload = require("./multerConfig")
+
+
+router.get('/', function (req, res) {
```

Comment:
We shouldn't have any routes that are returning a view. That is being handled front end via react.
Please remove this route